index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <h1 id="title"> survey form for microverse </h1>
+<P id="description">This is a test for the trials</P> 
```

Comment:
The p tag should be in lowercase to follow good practices.

Reply:
♻️ Good catch there! However, it is always a good idea to highlight the efforts the dev did while working on the code and then, ask for the changes you think are pertinent. Also, you could explain a bit more why it is a good practice to use only lowercase letters on HTML tags.